Jenkinsfile.stream

```diff
@@ -0,0 +1,63 @@
+def utils, streams
+node {
+    checkout scm
+    utils = load("utils.groovy")
+    streams = load("streams.groovy")
+    pod = readFile(file: "manifests/pod.yaml")
+}
+
+properties([
+    pipelineTriggers([]),
+    parameters([
+      choice(name: 'STREAM',
+             // list devel first so that it's the default choice
+             choices: (streams.production),
+             description: 'Fedora CoreOS stream metadata',
+             required: true),
+      string(name: 'VERSION',
+             description: 'Fedora CoreOS version getting released',
+             defaultValue: '',
+             trim: true)
+    ])
+])
+
+currentBuild.description = "[${params.STREAM}] Running"
+
+pod = pod.replace("COREOS_ASSEMBLER_IMAGE", "coreos-assembler:master")
+
+podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultContainer: 'jnlp') {
+    node('coreos-assembler') { container('coreos-assembler') {
+        stage('Stream Metadata' ) {
+            def gopath = "${env.WORKSPACE}" + "/go/"
+            def gocache = "${env.WORKSPACE}" + "/gocache/"
+            def coreosbot_token = utils.shwrap_capture("cat /.github/token")
```

Comment:
Just making sure, this `utils.shwrap_capture` doesn't show it's output correct?

Reply:
yeah, it doesn't show the command execution output. It basically return the result which is used further by coreosbot_token